src/ResourceManagement/DataLake.Analytics/Microsoft.Azure.Management.DataLake.Analytics/Customizations/DataLakeAnalyticsCustomizationHelper.cs

```diff
@@ -18,7 +18,7 @@ internal static class DataLakeAnalyticsCustomizationHelper
         /// This constant is used as the default package version to place in the user agent.
```

Comment:
@begoldsm the license header at the top of this file says the AutoRest version used is 0.14.0.0 rather than 1.0.0.0